File: ProiectCs/MPP/MPP/model/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace MPP.model
{
    public class Ticket : Identifiable<int>
    {
        public int Id { get; set; }

        public int Quantity
        {
            get;
            set;
        }

        public Match TicketMatch
        {
            get;
            set;
        }

        public string Name
        {
            get;
            set;
        }

        public Ticket()
        {
        }
        
        public Ticket(int id)
        {
            Id = id;
        }

        public Ticket(int id,int quantity, Match ticketMatch, string name)
        {
            Id = id;
            Quantity = quantity;
            TicketMatch = ticketMatch;
            Name = name;
        }

        public Ticket(int quantity, Match ticketMatch, string name)
        {
            Quantity = quantity;
            TicketMatch = ticketMatch;
            Name = name;
        }

        public override string ToString()
        {
            return Id + " " + Name + " " + TicketMatch + " " + Quantity;
        }
    }
}

File: JavaSockets/BaskModel/src/main/java/bask/model/Ticket.java
package bask.model;

import java.io.Serializable;

public class Ticket implements Identifiable<Integer>, Serializable {
    private Integer ID;
    private Integer quantity;
    private Match match;
    private String customerName;

    public Ticket() {

    }

    public Integer getQuantity() {
        return quantity;
    }

    public void setQuantity(Integer quantity) {
        this.quantity = quantity;
    }

    public Match getMatch() {
        return match;
    }

    public void setMatch(Match match) {
        this.match = match;
    }

    public Ticket(Integer quantity, Match match, String customerName) {
        this.quantity = quantity;
        this.match = match;
        this.customerName = customerName;
    }

    public Ticket(Integer ID, Integer quantity, Match match, String customerName) {
        this.ID = ID;
        this.quantity = quantity;
        this.match = match;
        this.customerName = customerName;
    }

    public Ticket(Integer ID) {
        this.ID = ID;
    }

    public String getCustomerName() {
        return customerName;
    }

    public void setCustomerName(String customerName) {
        this.customerName = customerName;
    }

    @Override
    public Integer getId() {
        return ID;
    }

    @Override
    public void setId(Integer id) {
        ID = id;
    }

    @Override
    public String toString() {
        return ID + " " +  customerName + " " + quantity + " " + match.toString();
    }
}
